### PR TITLE
minimal dockerfile to build and run ToxPipe

### LINKED ISSUE #1: Dockerize ChemCrow

### PR DESCRIPTION
**Pull request by vestalisvirginis**
closes #1

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,10 +5,16 @@ WORKDIR /usr/src
...
-COPY requirements.txt .
+RUN git clone https://github.com/NCBI-Codeathons/mlxai-2024-team-saddler.git
+
+#COPY requirements.txt .
 
 RUN pip install --upgrade pip
 RUN pip install wheel setuptools build
-RUN pip install -r requirements.txt
+#RUN pip install -r requirements.txt
+RUN pip install ipython langchain langchain_core langchain-openai molbloom nest_asyncio openai pandas python-dotenv rdkit rmrkl rxn4chemistry streamlit synspace tiktoken
```
Comment (boratyng) [inline Dockerfile:15]: Do we need `ipython`, `synspace`, and `streamlit`? I do not see them imported anywhere.

Comment (vestalisvirginis) [reply Dockerfile:15]: I kept ipython for now for dev purposes.
For synspace, good point. I will check it out.
Streamlit is needed for the interface that Trey is currently re-writing/adapting from ChemCrow

Comment (vestalisvirginis) [reply Dockerfile:15]: I am unable to check it for now because of the issue with 'SMILES Parse Error'.

Comment (combspk) [reply Dockerfile:15]: I'm trying to resolve this issue now.

**Review by combspk — APPROVED** [review]

**Pull request merged**